flutter_inappwebview_ios/ios/Resources/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you please let me know how to test this PR?

Reply:
Unfortunately, there's no real way to test this. 

The only way I can think of, is to build an app that just uses this as a dependency and submit it to review in App Store Connect and see whether you get an email complaining about this